event/manager.go

```diff
@@ -233,16 +237,15 @@ func (el *List) loadListFromEtcd(etcd *etcd.Client) {
 		event, err := loadEventFromJSON(jsonEvent)
 		if err != nil {
 			log.Errorf("load event from etcd err: %v", err)
-			return
+			continue
 		}
 		log.Infof("load event to list: %s", event.EventID)
 		w, err := LoadWorker(&event)
-		if err != nil {
```

Comment:
yes, i will delete "w, err := LoadWorker(&event) ". Here ie not need to do "LoadWorker".